tests/helpers/intl-test-helper.tsx

```diff
@@ -0,0 +1,121 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React, {
+    ReactElement,
+    ExoticComponent,
+    ForwardRefExoticComponent,
+} from 'react';
+import {
+    createIntl,
+    injectIntl,
+    IntlShape,
+    IntlProvider,
+} from 'react-intl';
+import {
+    shallow,
+    mount,
+    ShallowRendererProps,
+    MountRendererProps,
+} from 'enzyme';
+
+import defaultMessages from 'i18n/en.json';
+
+export const defaultIntl = createIntl({
+    locale: 'en',
+    defaultLocale: 'en',
+    timeZone: 'Etc/UTC',
+    messages: defaultMessages,
+    textComponent: 'span',
+});
+
+function unwrapForwardRef<WrappedComponentElement extends ReactElement>(element: ReactElement): WrappedComponentElement {
```

Comment:
This file could use some comments explaining what's going on here since it seems like we're having to dig into some React internals to get this function working with TypeScript.

If I understand this correctly, it takes a rendered component wrapped with forwardRef like `React.forwardRef(MyComponent)` and renders it as a `MyComponent`?

Reply:
correct.

Reply:
> This file could use some comments explaining what's going on here since it seems like we're having to dig into some React internals to get this function working with TypeScript.

I added some comments using TSDoc in recent commit.

Reply:
Awesome, thanks!